docs/security/access-control.rst

```diff
@@ -134,51 +123,106 @@ Permissions (each consistent of a resource + action pair) are then added to role
+/dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances                                  GET    DAGs.can_read, DAG Runs.can_read, Tasks.can_read                  Viewer
+/dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}                        GET    DAGs.can_read, DAG Runs.can_read, Tasks.can_read                  Viewer
+/dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}/links                  GET    DAGs.can_read, DAG Runs.can_read, Tasks.can_read                  Viewer
+/dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}/logs/{task_try_number} GET    DAGs.can_read, DAG Runs.can_read, Tasks.can_read                  Viewer
+/dags/~/dagRuns/~/taskInstances/list                                               POST   DAGs.can_read, DAG Runs.can_read, Tasks.can_read                  Viewer
+/variables                                                                         GET    Variables.can_read                                                Op
+/variables                                                                         POST   Variables.can_create                                              Op
+/variables/{variable_key}                                                          DELETE Variables.can_delete                                              Op
+/variables/{variable_key}                                                          GET    Variables.can_read                                                Op
+/variables/{variable_key}                                                          PATCH  Variables.can_edit                                                Op
+/dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}/xcomEntries            GET    DAGs.can_read, DAG Runs.can_read, Tasks.can_read, XComs.can_read  Viewer
+/dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}/xcomEntries/{xcom_key} GET    DAGs.can_read, DAG Runs.can_read, Tasks.can_read, XComs.can_read  Viewer
+================================================================================== ====== ================================================================= ============
+
+
+====================================== ======================================================================= ============
+Website Permissions
+-------------------------------------- ------------------------------------------------------------------------------------
+Action                                 Permissions                                                             Minimum Role
+====================================== ======================================================================= ============
+Access homepage                        Website.can_read                                                        Viewer
+Get DAG stats                          Dags.can_read, DAG Runs.can_read                                        Viewer
```

Comment:
And _somewhere_ you need to mention that either you need "Dags.can_read or the can_read permission to a specific dag"

Reply:
Added here https://github.com/apache/airflow/pull/11362/files#diff-5821e749f4956a14f3d0aa939985a5422ab9724bc11184244b47f620dce435f6R131